server/metrics.go

```diff
@@ -24,7 +24,13 @@ var (
 			Name:      "time_jump_back_total",
 			Help:      "Counter of system time jumps backward.",
 		})
-
+	bucketReportCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pd",
+			Subsystem: "server",
+			Name:      "bucket_report",
+			Help:      "Counter of bucket report.",
+		}, []string{"type", "status"})
```

Comment:
need store?

Reply:
the request of the stream does not container the information of the region leader or store id,  it does not need to distinguish the report count by store.

Reply:
Then do we know if the buckets are from one store or different stores?

Reply:
fix